packages/api/server/services/userNavigationLogs/insert.ts

```diff
@@ -0,0 +1,25 @@
+import userNavigationLogsModel from '#server/models/userNavigationLogsModel';
+const ServiceError = require('#server/errors/ServiceError');
+
+
+/**
+ * @param fk_user Identifiant de l'utilisateur connecté 
+ * @param page Page consultée
+ *
+ * @returns identifiant de la ligne insérée dans la table user_navigation_logs
+ */
```

Comment:
Pas besoin de jsdoc normalement avec le typing typescript. Ce dernier est censé remplacer le premier. Si vraiment la documentation (l'explication des paramètres) est indispensable, on peut déclarer un type Typescript dans lequel on vient faire ce détail.
Ici ça me paraître dispensable car très compréhensible en soi.

Reply:
Ok, c'est noté.